gradlew

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env sh
```

Comment:
Are these two scripts needed to run Gradle ? It seems optional according to Gradle's documentation.
Does the Android SDK come with Gradle ?